modules/orchestrator/con-environment-variables-supported-by-the-build-script.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="con-environment-variables-supported-by-the-build-script.adoc_{context}"]
+= Environment variables supported by the build script
+
+The `build-sh` script supports the following environment variables to customize the workflow build process without modifying the script itself.
+
+* `QUARKUS_EXTENSIONS`
+
+The `QUARKUS_EXTENSIONS` variable specifies additional Quarkus extensions required by the workflow.
+
+This variable takes the format of a comma-separated list of fully qualified extension IDs as shown in the following example:
+
+[source,yaml]
+----
+export QUARKUS_EXTENSIONS="io.quarkus:quarkus-smallrye-reactive-messaging-kafka"
+----
+
+Add Kafka messaging support or other integrations at build time.
+
+* `MAVEN_ARGS_APPEND`
+
+The `MAVEN_ARGS_APPEND` variable appends additional arguments to the Maven build command.
```

Comment:
@gfarache do you think we could add how to configure build process for disconnected envs?

Reply:
What do you mean by that? The disconnected envs already have all dependencies loaded in the image (the QUARKUS_EXTENTIONS) but the build conf can be customised I think

or do you mean updating the build script to use disconnected env? I think it's already possible if the user provide its own Dockerfile in which it uses the disconnected env images